cfme/tests/cloud/test_keypairs.py

```diff
@@ -46,9 +44,6 @@ def test_keypair_crud(openstack_provider):
         openstack_provider.mgmt.api.keypairs.delete(keypair.name)
         pytest.fail('Timed out deleting keypair')
 
-    with pytest.raises(KeyPairNotFound):
```

Comment:
This test assertion needs to be replaced somehow if the exception is no longer raised. I think the exception should still be raised and this block can stay as-is if so.